packages/sdk/src/protocols/filter.ts

```diff
@@ -56,11 +57,11 @@ export class SubscriptionManager implements ISubscriptionSDK {
 
   constructor(
     pubsubTopic: PubsubTopic,
-    remotePeers: Peer[],
+    getPeers: () => Peer[],
```

Comment:
why this change? I see later you just do `() => peers`

Reply:
hm interesting, I am not sure why it differs - as I believe it should refer to the same array